From rdunlap@xenotime.net  Fri May  1 13:51:53 2009
From: Randy Dunlap <rdunlap@xenotime.net>
Date: Wed, 29 Apr 2009 21:02:49 -0700
Subject: USB: usb.h: change private: kernel-doc for new format requirement
Cc: Greg KH <gregkh@suse.de>, David Brownell <dbrownell@users.sourceforge.net>
Message-ID: <49F922E9.6000901@xenotime.net>


From: Randy Dunlap <randy.dunlap@oracle.com>

Use "/* private:" to mark struct members as private so that
scripts/kernel-doc will handle them correctly.

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Acked-by: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/usb.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/include/linux/usb.h
+++ b/include/linux/usb.h
@@ -1424,8 +1424,8 @@ struct usb_sg_request {
 	int			status;
 	size_t			bytes;
 
-	/*
-	 * members below are private: to usbcore,
+	/* private:
+	 * members below are private to usbcore,
 	 * and are not provided for driver access!
 	 */
 	spinlock_t		lock;
